src/Microsoft.Health.Fhir.Shared.Api/Registration/FhirServerServiceCollectionExtensions.cs

```diff
@@ -58,6 +57,17 @@ public static class FhirServerServiceCollectionExtensions
                 .AddNewtonsoftJson()
                 .AddRazorRuntimeCompilation();
 
+            services.AddCors(options =>
```

Comment:
Could you add a comment explaining what this is doing? I think this is for the Banana Cake Pop UI, but I'm not certain.

Reply:
I changed it originally for my REACT-APP, using Banana Cake Pop UI doesn't need to add something. I restore the file to the original in the main branch.